theme/static/style/style.css

```diff
@@ -248,76 +248,93 @@ pre code {
 
 /* Screenshots Section */
 
-.screenshot-title{
+/* Title */
+.screenshot-title {
```

Comment:
In the future please make style changes in a separate commit or PR when not directly related to the current feature.